from core.exceptions.base_core_exception import BaseCoreException


class InvalidArgumentClassException(BaseCoreException):
    pass
